### PR TITLE
Fixed warnings, Fixed "run-built", added "build-react"

### PR DESCRIPTION
**Pull request by OmniacDev**
Fixed a bunch of ESLint warnings.
Fixed the "run-built" command. (previously threw error that "." was not a recognized command)
Added "build-react" command to quickly compile without electron. (to check for warnings etc)

### CODE REVIEW

File: src/pages/ModsPage.tsx [hunk 1]
```diff
@@ -36,8 +36,8 @@ function getAllMods(): ModErrorInfo[] {
 
         else {
             try {
-                const configData = fs.readFileSync(modConfigPath, "utf-8");
-                const configParsed = JSON.parse(configData);
```
Comment (FrederoxDev) [inline src/pages/ModsPage.tsx:40]: These lines do have a purpose to validate the json is not completely invalid though the try catch

**Review by FrederoxDev — APPROVED** [review]

File: src/pages/ModsPage.tsx (hunk 1 above)
Comment (FrederoxDev) [reply src/pages/ModsPage.tsx:40]: I will just re-add em

**Comment by FrederoxDev** [comment]
Seems to have issues with running on my machine

**Comment by OmniacDev** [comment]
what's up with it?

**Comment by FrederoxDev** [comment]
there was a hard crash on start, with a very non descriptive error. didn't look far into it since I want to work on the launcher this morning

**Comment by OmniacDev** [comment]
huh that's weird. was that after building?

 I did update one of the node modules, not sure if that would cause a crash though.

**Pull request merged**